zproject/prod_settings_template.py

```diff
@@ -76,8 +76,8 @@
 EMAIL_USE_TLS = True
 # The noreply address to be used as the sender for certain generated emails.
 # Messages sent to this address could contain sensitive user data and should
-# not be delivered anywhere.
-NOREPLY_EMAIL_ADDRESS = "Zulip <noreply@example.com>"
+# not be delivered anywhere. (e.g. "Zulip <noreply@example.com>")
+NOREPLY_EMAIL_ADDRESS = "Zulip <noreply@{0}>".format(EXTERNAL_HOST.split(":")[0])
```

Comment:
I think maybe using the `%` syntax would be clearer for people who aren't Python-literate reading this configuration file?  Or even `+` ?  Something to consider.